controllers/dtpullsecret/generate.go

```diff
@@ -83,5 +83,5 @@ func pullSecretDataFromDockerConfig(dockerConf *dockerConfig) (map[string][]byte
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
-	return map[string][]byte{".dockerconfigjson": dockerConfJson}, nil
+	return map[string][]byte{dockerConfigJson: dockerConfJson}, nil
```

Comment:
Are you certain that this works?

Reply:
Yes, I am. Just replacing a string with a constant of the same string value